fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -177,7 +177,10 @@ public void auditAfterExec(String origStmt, StatementBase parsedStmt, PQueryStat
 
         ctx.getAuditEventBuilder().setFeIp(FrontendOptions.getLocalHostAddress());
 
-        if (ctx.getState().isQuery() && containsComment(origStmt)) {
+        if (!ctx.getState().isQuery() && (parsedStmt != null && parsedStmt.needAuditEncryption())) {
+            // Some information like username, password in the stmt should not be printed.
+            ctx.getAuditEventBuilder().setStmt(parsedStmt.toString());
```

Comment:
toString is an incomplete interface. Unified to sql functionality in AST2SQL

Reply:
OK. It has been replaced by `AST2SQL.toString(parsedStmt)`.